gym/wrappers/record_episode_statistics.py

```diff
@@ -0,0 +1,34 @@
+import time
+from collections import deque
+
+import gym
+
+
+class RecordEpisodeStatistics(gym.Wrapper):
+    def __init__(self, env, deque_size=100):
+        super(RecordEpisodeStatistics, self).__init__(env)
+        self.t0 = time.time()  # TODO: use perf_counter when gym removes Python 2 support
+        self.episode_return = 0.0
+        self.episode_horizon = 0
```

Comment:
would episode_length not be more descriptive?

Reply:
@pzhokhov Indeed, it is more descriptive. Thanks!